test/suites/optional/healthchecks-disabled-service.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Documentation       Test if healthcheck exits quickly when MicroShift service is disabled
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/microshift-process.resource
+Library             DateTime
+
+Suite Setup         Setup Suite
+Suite Teardown      Teardown Suite
+
+
+*** Test Cases ***
+Healthchecks Should Exit Fast And Successful When MicroShift Is Disabled
+    [Documentation]    When microshift.service is disabled, the healtchecks should
+    ...    exit quickly and without an error. They should not attempt to query API Server.
+    [Tags]    ushift-5381
+    [Setup]    Disable MicroShift
+    ${before}=    Get Current Date
+    ${stdout}    ${stderr}    ${rc}=    SSHLibrary.Execute Command
+    ...    /etc/greenboot/check/required.d/41_microshift_running_check_multus.sh
```

Comment:
Should we do this for all the scripts in that directory except the main one?

Reply:
Done